{"uuid": "4f99159c-fb91-4d91-91e9-2d7f00208eb1", "children": ["9eaecc89-3e17-4ab3-98f5-e5f50d8c8162"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/Users/opk/otus/develop/web_hw/~/otus/develop/web_hw/logs/test_check_number_of_output_elements.log'\n", "trace": "  File \"/Users/opk/otus/develop/web_hw/venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/opk/otus/develop/web_hw/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/opk/otus/develop/web_hw/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/opk/otus/develop/web_hw/conftest.py\", line 95, in driver\n    logger = setup_logging(log_level, test_name)\n  File \"/Users/opk/otus/develop/web_hw/utils.py\", line 5, in setup_logging\n    logger_handler = logging.FileHandler(f\"~/otus/develop/web_hw/logs/{test_name}.log\", mode=\"a\")\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/logging/__init__.py\", line 1146, in __init__\n    StreamHandler.__init__(self, self._open())\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/logging/__init__.py\", line 1175, in _open\n    return open(self.baseFilename, self.mode, encoding=self.encoding,\n"}, "start": 1658902783230, "stop": 1658902783230}], "start": 1658902783230, "stop": 1658902783290}